### PR TITLE
Km slide seq doc updates

### PR DESCRIPTION
**Pull request by kayleemathews**
### Description

Updated Slide-seq documentation to address suggested changes and match the current version of the pipeline.

----

### Checklist 
If you can answer "yes" to the following items, please add a checkmark next to the appropriate checklist item(s) **and** notify our WARP documentation team by tagging either @ekiernan or @kayleemathews in a comment on this PR.

- [ ] Did you add inputs, outputs, or tasks to a workflow?
- [ ] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [ ] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

**Review by jessicaway — APPROVED** [review]
👍 thanks @kayleemathews !

File: website/docs/Pipelines/SlideSeq_Pipeline/README.md [hunk 1]
```diff
@@ -36,83 +40,83 @@ The following table provides a quick glance at the SlideSeq pipeline features:
...
-
-The SlideSeq pipeline can be deployed using [Cromwell](https://cromwell.readthedocs.io/en/stable/), a GA4GH compliant, flexible workflow management system that supports multiple computing platforms. The workflow can also be run in [Terra](https://app.terra.bio), a cloud-based analysis platform. 
+If you’re running a Slide-seq workflow version prior to the latest release, the accompanying documentation for that release may be downloaded with the source code on the WARP [releases page](https://github.com/broadinstitute/warp/releases) (see the source code folder `website/pipelines/skylab/slideseq`.
 
-<!--- link to public workspace will go here --->
+The Slide-seq pipeline can be deployed using [Cromwell](https://cromwell.readthedocs.io/en/stable/), a GA4GH compliant, flexible workflow management system that supports multiple computing platforms. The workflow can also be run in [Terra](https://app.terra.bio), a cloud-based analysis platform. The [Slide-seq Public Workspace](https://app.terra.bio/#workspaces/warp-pipelines/Slide-seq) on Terra contains the Slide-seq workflow, workflow configuration, required reference data and other inputs, and example testing data.
```
Comment (ekiernan) [inline website/docs/Pipelines/SlideSeq_Pipeline/README.md:51]: Thank you for adding this!

File: website/docs/Pipelines/SlideSeq_Pipeline/README.md [hunk 2]
```diff
@@ -36,83 +40,83 @@ The following table provides a quick glance at the SlideSeq pipeline features:
...
+The example Slide-seq reference files are created using the [BuildIndices pipeline](https://github.com/broadinstitute/warp/blob/master/pipelines/skylab/build_indices/BuildIndices.wdl). The BuildIndices pipeline modifies a given GTF downloaded from GENCODE to only include biotypes that are listed in a tab separated file ([biotypes.tsv](https://github.com/broadinstitute/warp/blob/master/dockers/skylab/snss2-build-indices/Biotypes.tsv)). The example references do not include the pseudogene biotype. Learn more about Ensembl biotypes in the [Ensembl overview](https://m.ensembl.org/info/genome/genebuild/biotypes.html).
 
-<!--- #### Sample inputs for analyses in a Terra Workspace --->
+#### Sample inputs for analyses in a Terra Workspace
 
-## SlideSeq tasks and tools
+The Slide-seq pipeline is currently available on the cloud-based platform Terra. After registering, you can access the [Slide-seq public workspace](https://app.terra.bio/#workspaces/warp-pipelines/Slide-seq) which is preloaded with instructions and sample data. Please view the [Support Center](https://support.terra.bio/hc/en-us) for more information on using the Terra platform.
```
Comment (ekiernan) [inline website/docs/Pipelines/SlideSeq_Pipeline/README.md:78]: You called it the "Slide-seq Public Workspace" in the first mention. Maybe change the first instance to lower case like this?

Comment (kayleemathews) [reply website/docs/Pipelines/SlideSeq_Pipeline/README.md:78]: Good catch Liz!

**Pull request merged**